docs/source/yelpsoa_configs.rst

```diff
@@ -279,6 +279,14 @@ Here is an example smartstack.yaml::
       proxy_port: 20028
       timeout_server_ms: 5000
 
+The main key is the service namespace.  Namespaces were introduced for PaaSTA
```

Comment:
Can you do ``main`` if the intention here reference that string in the config file? At first I thought you meant "main" as in primary